casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -56,6 +60,31 @@ class RuntimeManager private (val initStateHash: ByteString, runtimeContainer: S
     result
   }
 
+  def computeBonds(hash: StateHash): Seq[Bond] = {
+    val resetRuntime     = getResetRuntime(hash)
+    val bondsChannel     = Channel(Quote(Par().copy(exprs = Seq(Expr(GString("proofOfStake"))))))
```

Comment:
I'm concerned about this import source of truth just living at a public name. I would prefer if it lived at an unforgeable name inside a contract and there was read-only access to it via a public name.

As things are today, any validator could unbond any other validator by just changing the bonds listing, however I think that validators should need to unbond themselves through a message they sign. We could get that kind of security through the PoS contract if the bond listing is maintained internally.

Reply:
I'm not quite aware of how to build a read-only name from protobuf-generated case classes. Would a 

// TODO:  Switch to a read only name

do for now?